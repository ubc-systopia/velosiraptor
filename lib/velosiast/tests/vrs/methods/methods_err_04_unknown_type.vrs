// wrong type from parameter
fn foo(foo: int, bar: UNKNOWN) -> int
{
    1
}
