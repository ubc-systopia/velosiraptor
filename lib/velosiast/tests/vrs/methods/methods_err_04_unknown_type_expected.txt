error: no symbol named `UNKNOWN` found in the current scope
     --> tests/vrs/methods/methods_err_04_unknown_type.vrs:2:23
      |
    2 |  fn foo(foo: int, bar: UNKNOWN) -> int
      |                        ^^^^^^^ not found in this scope
error: could not compile due to 1 previous errors; 0 warnings emitted


@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@

